frontend/src/components/accounts/CreateAccount.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+let inputAmount;
+
+class CreateAccount extends React.Component {
+  constructor() {
+    super()
+    
+    this.state = {
+      value: 'USD',
+      checked: false,
+      status: 'emptyNot'
+    }
+
+    this.handleCheckButton = this.handleCheckButton.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  render() {
+    return (
+      <div>
+        <button className="btn btn-default btn-md" 
+                onClick={this.handleCheckButton} 
+                checked={this.state.checked}>
+          Create New Account
+        </button>
+        {this.toggleDiv()}
+      </div>
+    );
+  }
+
+  toggleDiv() {
```

Comment:
toggleDiv doesn't sounds good as it just returns a HTML content which to be rendered. 

```javascript
const newAccountForm = renderCreateAccountForm()
{newAccountForm} 
```